plugins/Marketplace/resources/qml/OnboardBanner.qml

```diff
@@ -0,0 +1,138 @@
+// Copyright (c) 2021 Ultimaker B.V.
+// Cura is released under the terms of the LGPLv3 or higher.
+
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.1
+
+import UM 1.6 as UM
+import Cura 1.6 as Cura
+
+// Onboarding banner.
+Rectangle
+{
+    property bool bannerVisible
+    property string bannerIcon
+    property string bannerBody
+    property var onRemoveBanner
+    property string readMoreUrl
+
+    visible: bannerVisible
+
+    Layout.preferredHeight: childrenRect.height + 2 * UM.Theme.getSize("default_margin").height
+    anchors
+    {
+        margins: UM.Theme.getSize("default_margin").width
+        left: parent.left
+        right: parent.right
+        top: parent.top
+    }
+
+    color: UM.Theme.getColor("action_panel_secondary")
+
+    // Icon
+    Rectangle
+    {
+        id: onboardingIcon
+        anchors
+        {
+            top: parent.top
+            left: parent.left
+            margins: UM.Theme.getSize("default_margin").width
+        }
+        width: UM.Theme.getSize("button_icon").width
+        height: UM.Theme.getSize("button_icon").height
+        color: "transparent"
+        UM.RecolorImage
+        {
+            anchors.fill: parent
+            color: UM.Theme.getColor("primary_text")
+            source: UM.Theme.getIcon(bannerIcon)
+        }
+    }
+
+    // Close button
+    UM.SimpleButton
+    {
+        id: onboardingClose
+        anchors
+        {
+            top: parent.top
+            right: parent.right
+            margins: UM.Theme.getSize("default_margin").width
+        }
+        width: UM.Theme.getSize("message_close").width
+        height: UM.Theme.getSize("message_close").height
+        color: UM.Theme.getColor("primary_text")
+        hoverColor: UM.Theme.getColor("primary_text_hover")
+        iconSource: UM.Theme.getIcon("Cancel")
+
+        onClicked: onRemoveBanner()
+    }
+
+    // Body
+    Label {
+        id: infoText
+        anchors
+        {
+            top: parent.top
+            left: onboardingIcon.right
+            right: onboardingClose.left
+            margins: UM.Theme.getSize("default_margin").width
+        }
+
+        font: UM.Theme.getFont("default")
+        text: bannerBody
+
+        renderType: Text.NativeRendering
+        color: "white"
```

Comment:
Hardcoded value, needs to be pulled from the theme

Reply:
https://github.com/Ultimaker/Cura/pull/10948/commits/6909085733a46bda1e99fe36c2eaba07ae6ed32d